cli/bblfshctl/main.go

```diff
@@ -16,17 +16,49 @@ var (
 
 func main() {
 	parser := flags.NewNamedParser("bblfshctl", flags.Default)
-	parser.AddCommand("status", cmd.StatusCommandDescription, "", &cmd.StatusCommand{})
-	parser.AddCommand("instances", cmd.InstancesCommandDescription, "", &cmd.InstancesCommand{})
-	parser.AddCommand("drivers", cmd.DriversCommandDescription, "", &cmd.DriversCommand{})
-	parser.AddCommand("parse", cmd.ParseCommandDescription, "", &cmd.ParseCommand{})
+	parser.AddCommand("status",
+		cmd.StatusCommandDescription, cmd.StatusCommandHelp,
+		&cmd.StatusCommand{},
+	)
+
+	parser.AddCommand("instances",
+		cmd.InstancesCommandDescription, cmd.InstancesCommandHelp,
+		&cmd.InstancesCommand{},
+	)
+
+	parser.AddCommand("parse",
+		cmd.ParseCommandDescription, cmd.ParseCommandHelp,
+		&cmd.ParseCommand{},
+	)
+
+	c, _ := parser.AddCommand("driver",
+		cmd.DriverCommandDescription, cmd.DriverCommandHelp,
+		&cmd.DriverCommand{},
+	)
+
+	c.AddCommand("list",
+		cmd.DriverListCommandDescription, cmd.DriverListCommandHelp,
+		&cmd.DriverListCommand{},
+	)
+
+	c.AddCommand("install",
+		cmd.DriverInstallCommandDescription, cmd.DriverInstallCommandHelp,
+		&cmd.DriverInstallCommand{},
+	)
+
+	c.AddCommand("remove",
+		cmd.DriverRemoveCommandDescription, cmd.DriverRemoveCommandHelp,
+		&cmd.DriverRemoveCommand{},
+	)
 
 	if _, err := parser.Parse(); err != nil {
-		if _, ok := err.(*flags.Error); ok {
+		if flagsErr, ok := err.(*flags.Error); ok && flagsErr.Type == flags.ErrHelp {
+			os.Exit(0)
+		} else {
+			fmt.Println()
 			parser.WriteHelp(os.Stdout)
-			fmt.Printf("\nBuild information\n  commit: %s\n  date: %s\n", version, build)
+			fmt.Printf("\nBuild information\n  commit: %s\n  date:%s\n", version, build)
```

Comment:
Why is this line changed?

Reply:
the missing space

Reply:
I see there's now a missing space, I don't see the point of removing that space, that's what I was asking.

Reply:
because you have a space on commit, so you should have one in the date too. if not looks wrong

Reply:
You're removing the space, not adding it.